requirements.txt

```diff
@@ -1,5 +1,5 @@
 django>=1.11,<2.2
 django-leaflet>=0.23,<0.25
 channels>=2.1.3,<2.4.0
-Pillow>=4.3.0,<6.2
+Pillow>=6.0.0,<=6.3.0
```

Comment:
let's take advantage to force the upgrade of pillow since it seems there's a vulnerability in the previous version, use:

`Pillow>=6.2.0,<=6.3.0`